matlab/test/arrow/io/ipc/tRecordBatchStreamReader.m

```diff
@@ -0,0 +1,336 @@
+            writer = arrow.io.ipc.RecordBatchStreamWriter(fname, recordBatch1.Schema);
+            writer.writeRecordBatch(recordBatch1);
+            writer.writeRecordBatch(recordBatch2);
+            writer.close();
+            testCase.MultipleBatchStreamFile = fname;
+        end
+    end
+
+    methods (Test)
+
+        function ZeroLengthFilenameError(testCase)
+            % Verify RecordBatchStreamReader throws an exception with the
+            % identifier MATLAB:validators:mustBeNonzeroLengthText if the
+            % filename input argument given is a zero length string.
+            fcn = @() arrow.io.ipc.RecordBatchStreamReader("");
+            testCase.verifyError(fcn, "MATLAB:validators:mustBeNonzeroLengthText");
+        end
+
+        function MissingStringFilenameError(testCase)
+            % Verify RecordBatchStreamReader throws an exception with the
+            % identifier MATLAB:validators:mustBeNonzeroLengthText if the
+            % filename input argument given is a missing string.
+            fcn = @() arrow.io.ipc.RecordBatchStreamReader(string(missing));
+            testCase.verifyError(fcn, "MATLAB:validators:mustBeNonzeroLengthText");
+        end
+
+        function FilenameInvalidTypeError(testCase)
+            % Verify RecordBatchStreamReader throws an exception with the
+            % identifier MATLAB:validators:UnableToConvert if the filename
+            % input argument is neither a scalar string nor a char vector.
+            fcn = @() arrow.io.ipc.RecordBatchStreamReader(table);
+            testCase.verifyError(fcn, "MATLAB:validation:UnableToConvert");
+        end
+
+        function Schema(testCase)
+            % Verify the getter method for Schema returns the
+            % expected value.
+            fieldA = arrow.field("A", arrow.string());
+            fieldB = arrow.field("B", arrow.float32());
+            expectedSchema = arrow.schema([fieldA fieldB]);
+
+            reader = arrow.io.ipc.RecordBatchStreamReader(testCase.ZeroBatchStreamFile);
+            testCase.verifyEqual(reader.Schema, expectedSchema);
+
+            reader = arrow.io.ipc.RecordBatchStreamReader(testCase.OneBatchStreamFile);
+            testCase.verifyEqual(reader.Schema, expectedSchema);
+
+            reader = arrow.io.ipc.RecordBatchStreamReader(testCase.MultipleBatchStreamFile);
+            testCase.verifyEqual(reader.Schema, expectedSchema);
+        end
+
+        function SchemaNoSetter(testCase)
+            % Verify the Schema property is not settable.
+            fieldC = arrow.field("C", arrow.date32());
+            schema = arrow.schema(fieldC);
+            reader = arrow.io.ipc.RecordBatchStreamReader(testCase.ZeroBatchStreamFile);
+            testCase.verifyError(@() setfield(reader, "Schema", schema), "MATLAB:class:SetProhibited");
+        end
+
+        function ReadZeroBatchStreamFile(testCase, RecordBatchReadFcn)
+            % Verify read can successfully read an Arrow IPC Stream file
```

Comment:
Can you update this comment to a bit more clear?

Reply:
Sorry. I think I meant to update this comment.

Reply:
Fixed.

Reply:
Fixed.